File: PortfolioApp.API/Models/Project.cs
using System;
using System.Collections.Generic;

namespace PortfolioApp.API.Models
{
    public class Project
    {
        public int Id { get; set; }       
        public string ProjectName { get; set; }
        public string  CompanyName { get; set; }
        public DateTime DateOn { get; set; }
        public DateTime DateOf { get; set; }
        public string Mission { get; set; }
        public string Environment {get;set;}
        public string City { get; set; }
        public string Country { get; set; }
        public ICollection<PhotoProjects> Photos  { get; set; }
    }
}

File: PortfolioApp-WEB/src/app/_models/project.ts
import { PhotoProject } from "./photoProject";

export interface Project {
    //--> Propriétés For List
    id:number;   
    projectName: string;
    photoUrl: string;

    //--> Propriétés For Details
    companyName?: string ;
    dateOn?: Date;
    dateOf?: Date;
    mission?: string;
    environment?: string ;
    city?: string ;
    country?: string;
    photos?:PhotoProject[];
}
